packages/client/src/bags/types/getBagOperation.types.ts

```diff
@@ -0,0 +1,9 @@
+import type { Bag } from './bag.types';
+import type { BagOperation } from './bagOperation.types';
+import type { Config } from '../..';
+
+export type GetBagOperation = (
+  id: Bag['id'] | null,
```

Comment:
I think this cannot be null, can you confirm please?

Reply:
That's right. Removed.